.eslintrc.json

```diff
@@ -53,6 +60,8 @@
     "react/prefer-stateless-function": 0,
     "react/sort-comp": 0,
     "jsx-a11y/media-has-caption": 0,
-    "no-underscore-dangle": 0
+    "no-underscore-dangle": 0,
+    "import/extensions": 0,
+    "react/default-props-match-prop-types": 0
```

Comment:
There are a bunch of issues with flow not doing normal things with defaultProps. It will work correctly, but this lint rule doesn't interpret the code correctly.
https://github.com/facebook/flow/issues/6882